Remote access docs

BKristenssonAlfsson opened the pull request:
Made a basic sequence diagram in how the flow is going through the different files and functions for connecting.

BKristenssonAlfsson committed:
docs: Documentation for connection UI to Edge

Describing how the user can connect to either local network,
or passing in a room ID to connect to a remote Edge

BKristenssonAlfsson committed:
docs: Removed duplicate images

Two images that was already present under different name.
Removed the newly inserted to keep the images that was
hiding the Peer ID

ivelin reviewed (commented):
@BKristenssonAlfsson would you like to update this document to reflect the latest behavior we just merged?

Also, you can link to this document from the docs menu in [this yaml](https://github.com/ambianic/ambianic-docs/blob/master/mkdocs.yml).

BKristenssonAlfsson committed:
fix: Updated documentation for remote connection

The edge connect document is updated to show the new flow
on how the UI is connecting to either a local network or to
a remote network.

BKristenssonAlfsson commented:
@ivelin Updated the documentation

ivelin reviewed (commented):
@BKristenssonAlfsson this is great. Thank you! Do you think it belongs in the Developer or the User section of the docs menu?

BKristenssonAlfsson committed:
fix: changed location of edge connect

As first iteration was a technical documentation,
this file stayed there when it was changed to an users documentation.
Moved file to right place and updated mkdocs.yml

Commit:
docs: replace connection with pairing

@BKristenssonAlfsson lets use pairing (bluetooth terminology) when we refer to initial connectivity and use connection when we refer to consecutive communication between UI and Edge.

ivelin commented on mkdocs.yml:
```diff
@@ -12,6 +12,7 @@ nav:
       - AI Apps: users/ai-apps.md
       - How To: users/howto.md
       - Support: users/support.md
+      - Remote and Local pairing: developers/edge-connection.md
```
@BKristenssonAlfsson I see the file is now referenced in the User section, which makes sense, but it resides in the developer directory. Is that the intention?

ivelin replied:
Have you tested this locally? The file is now in users directory, but the link in mkdocs.yml points to a file in the dev dir.

ivelin commented:
@BKristenssonAlfsson related to my earlier comment. Are you ok with renaming the file edge-pairing.md to clarify that we are talking about the initial connectivity setup and not accessing edge remotely in general.

BKristenssonAlfsson committed:
fix: renaming and updated link

Should be correct this time

BKristenssonAlfsson committed:
Merge branch 'master' into remote-access-docs

BKristenssonAlfsson committed:
Merge branch 'remote-access-docs' of https://github.com/BKristenssonAlfsson/ambianic-docs into remote-access-docs

Commit:
docs: stylistic edits
